public/app/core/components/Footer/Footer.tsx

```diff
@@ -11,23 +11,12 @@ export interface FooterLink {
 }
 
 export let getFooterLinks = (): FooterLink[] => {
+  // Fork: Replace Grafana links with link to our fork
   return [
     {
-      text: 'Documentation',
+      text: 'Open Source Fork',
```

Comment:
Nit: Fork is a GitHub specific term...maybe something like "Source Code" would be more clear